robot_params:
  qr_port: "/dev/ttyUSB0"
  qr_baud: 115200
  qr_timeout: 0.5
  qr_retry: 3
  qr_retry_delay: 0.5
  qr_retry_delay_max: 1.0
  qrs:
    Q1 : "1000 0"
    Q2 : "1750 0"
    Q3 : "3750 0"
    Q4 : "4375 0"
    Q5 : "6375 0"
    Q6 : "7000 0"
    Q7 : "8000 0"
    Q8 : "9000 0"
    Q9  : "9625 0"
    Q10 : "11625 0"
    Q11 : "12250 0"
    Q12 : "14250 0"
    Q13 : "15000 0"
    Q14 : "16000 1000"
    Q15 : "16000 11500"
    Q16 : "15000 12500"
    Q17 : "14250 12500"
    Q18 : "12250 12500"
    Q19 : "11625 12500"
    Q20 : "9625 12500"
    Q21 : "9000 12500"
    Q22 : "8000 12500"
    Q23 : "7000 12500"
    Q24 : "6375 12500"
    Q25 : "4375 12500"
    Q26 : "3750 12500"
    Q27 : "1750 12500"
    Q28 : "1000 12500"
    Q29 : "0 11500"
    Q30 : "0 1000"
    Q31 : "2750 1000"
    Q32 : "2750 5250"
    Q33 : "2750 6250"
    Q34 : "2750 7250"
    Q35 : "2750 11500"
    Q36 : "5375 11500"
    Q37 : "5375 7250"
    Q38 : "5375 6250"
    Q39 : "5375 5250"
    Q40 : "5375 1000"
    Q41 : "8000 1000"
    Q42 : "8000 11500"
    Q43 : "10625 11500"
    Q44 : "10625 7250"
    Q45 : "10625 6250"
    Q46 : "10625 5250"
    Q47 : "10625 1000"
    Q48 : "13250 1000"
    Q49 : "13250 5250"
    Q50 : "13250 6250"
    Q51 : "13250 7250"
    Q52 : "13250 11500"
  bosenaryolar:
    senaryo1 : 
     - senaryo : "S1-3-4-S2-1-2-S1"
     - qr : "Q22-Q21-Q20-Q19-Q18-Q17-Q16-Q15-Q14-Q13-Q12-Q11-Q10-Q9-Q8-Q6-Q5-Q4-Q3-Q2-Q1-Q30-Q29-Q28-Q27-Q26-Q25-Q24-Q22"
     - karar : "Q21-Q20-Q18-Q16-Q14-Q12-Q10-Q8-Q5-Q3-Q1-Q29-Q27-Q25-Q23-Q22"
     - hedef : "Q21-Q19-Q17-Q15-Q13-Q11-Q9-Q6-Q4-Q2-Q30-Q28-Q26-Q24-Q22-Q22"
    senaryo2 : 
      - senaryo : "S1-2-1-S2-4-3-S1"
      - qr : "Q24-Q25-Q26-Q27-Q28-Q29-Q30-Q1-Q2-Q3-Q4-Q5-Q6-Q8-Q9-Q10-Q11-Q12-Q13-Q14-Q15-Q16-Q17-Q18-Q19-Q20-Q22"
      - karar : "Q23-Q24-Q26-Q28-Q30-Q2-Q4-Q6-Q9-Q11-Q13-Q15-Q17-Q19-Q21-Q22"
      - hedef : "Q23-Q25-Q27-Q29-Q1-Q3-Q5-Q8-Q10-Q12-Q14-Q16-Q18-Q20-Q22-Q22"
    senaryo3 :
      - senaryo : "S2-1-2-S1-3-4-S2"
      - qr : "Q5-Q4-Q3-Q2-Q1-Q30-Q29-Q28-Q27-Q26-Q25-Q24-Q23-Q21-Q20-Q19-Q18-Q17-Q16-Q15-Q14-Q13-Q12-Q11-Q10-Q9-Q7"
      - karar : "Q6-Q5-Q3-Q1-Q29-Q27-Q25-Q23-Q20-Q18-Q16-Q14-Q12-Q10-Q8-Q7"
      - hedef : "Q6-Q4-Q2-Q30-Q28-Q26-Q24-Q21-Q19-Q17-Q15-Q13-Q11-Q9-Q7-Q7"
    senaryo4 :
      - senaryo : "S2-4-3-S1-2-1-S2"
      - qr : "Q9-Q10-Q11-Q12-Q13-Q14-Q15-Q16-Q17-Q18-Q19-Q20-Q21-Q23-Q24-Q25-Q26-Q27-Q28-Q29-Q30-Q1-Q2-Q3-Q4-Q5-Q6-Q7"
      - karar : "Q8-Q9-Q11-Q13-Q15-Q17-Q19-Q21-Q24-Q26-Q28-Q30-Q2-Q4-Q6-Q7"
      - hedef : "Q8-Q10-Q12-Q14-Q16-Q18-Q20-Q23-Q25-Q27-Q29-Q1-Q3-Q5-Q7-Q7"
  
  yuklusenaryolar:
    senaryo1 :
     - senaryo : "S1-F-S2-A-E-S1-C-D-G-S1-B-S2-F-S1"
     - qr : "Q42-Q41-Q8-Q9-Q10-Q11-Q48-Q49-Q50-Q52-Q18-Q19-Q20-Q21-Q23-Q24-Q36-Q37-Q38-Q40-Q4-Q3-Q31-Q32-Q33-Q35-Q26-Q25-Q24-Q23-Q21-Q20-Q43-Q44-Q45-Q47-Q9-Q8-Q41-Q42-Q22"
     - karar : "Q42-Q41-Q9-Q11-Q49-Q52-Q19-Q21-Q24-Q37-Q40-Q3-Q32-Q35-Q25-Q23-Q20-Q44-Q47-Q8-Q42-Q22"
     - hedef : "Q42-Q8-Q10-Q48-Q50-Q18-Q20-Q23-Q36-Q38-Q4-Q31-Q33-Q26-Q24-Q21-Q43-Q45-Q9-Q41-Q22-Q22"
     - yukistasyon :
         Q50 : HIGH
         Q38 : LOW
         Q33 : HÄ°GH
         Q45 : LOW
         
       
    senaryo2 :
     - senaryo : "S1-F-S2-B-S1-G-D-C-S1-E-A-S2-F-S1"
     - qr : "Q41-Q8-Q9-Q47-*46-45-44*-Q43-Q20-Q21-Q23-Q24-Q25-Q26-Q35-*Q34-Q33-Q32*-Q31-Q3-Q4-Q40-*Q39-Q38-Q37*-Q36-Q24-Q23-Q21-Q20-Q19-Q18-Q52-*Q51-Q50-Q49*-Q48-Q11-Q10-Q9-Q8-Q41-Q42-Q22"
     - karar : "Q42-Q41-Q9-Q46-Q43-Q21-Q24-Q26-Q34-Q31-Q4-Q39-Q36-Q23-Q20-Q18-Q51-Q48-Q10-Q8-Q42-Q22"
     - hedef : "Q42-Q8-Q47-Q45-Q20-Q23-Q25-Q35-Q33-Q3-Q40-Q38-Q24-Q21-Q19-Q52-Q50-Q11-Q9-Q41-Q22-Q22"
     - yukistasyon:
         Q45 : HIGH
         Q33 : LOW
         Q38 : HIGH
         Q50 : LOW
  
    senaryo3 :
      - senaryo : "S2-F-S1-G-D-S2-B-E-A-S2-C-S1-F-S2"
      - qr : "Q41-Q42-Q23-Q24-Q25-Q26-Q35-*Q34-Q33-Q32*-Q31-Q3-Q4-Q5-Q6-Q8-Q9-Q47-*Q46-Q45-Q44*-Q43-Q19-Q18-Q52-*Q51-Q50-Q49*-Q48-Q11-Q10-Q9-Q8-Q6-Q5-Q40-*Q39-Q38-Q37*-Q36-Q24-Q23-Q42-Q41-Q7"
      - karar : "Q41-Q42-Q24-Q26-Q34-Q31-Q4-Q6-Q9-Q46-Q43-Q18-Q51-Q48-Q10-Q8-Q5-Q39-Q36-Q23-Q41-Q7"
      - hedef : "Q41-Q23-Q25-Q35-Q33-Q3-Q5-Q8-Q47-Q45-Q19-Q52-Q50-Q11-Q9-Q6-Q40-Q38-Q24-Q42-Q7-Q7"
      - yukistasyon :
          Q33 : HIGH
          Q45 : LOW
          Q50 : HIGH
          Q38 : LOW
          
        
    senaryo4 :
      - senaryo : "S2-F-S1-C-S2-A-E-B-S2-D-G-S1-E-S2"
      - qr : "Q41-Q42-Q23-Q24-Q36-*Q37-Q38-Q39-Q40-Q5-Q6-Q8-Q9-Q10-Q11-Q48-*Q49-Q50-Q51*-Q52-Q18-Q19-Q43-*Q44-Q45-Q46*-Q47-Q9-Q8-Q6-Q5-Q4-Q3-Q31-*Q32-Q33-Q34*-Q35-Q26-Q25-Q24-Q23-Q42-Q41-Q7"
      - karar : "Q41-Q42-Q24-Q37-Q40-Q6-Q9-Q11-Q49-Q52-Q19-Q44-Q47-Q8-Q5-Q3-Q32-Q35-Q25-Q23-Q41-Q7"
      - hedef : "Q41-Q23-Q36-Q38-Q5-Q8-Q10-Q48-Q50-Q18-Q43-Q45-Q9-Q6-Q4-Q31-Q33-Q26-Q24-Q42-Q7-Q7"
      - yukistasyon :
          Q38: HIGH
          Q50: LOW
          Q45 : HIGH
          Q33 : LOW



  yukbosnokta : [A,B,C,D]
  kosenokta:  [1,2,3,4]
  engelnokta : [E,F,G]
  basbitisnokta : [S1,S2]
